src/needleman_wunch.jl

```diff
@@ -5,11 +5,14 @@
 
 Your documentation here.
 """
-function nwscore(base1::Char, base2::Char)
-    
-end
-
+function nwscore(base1::Char, base2::Char; match, mismatch)
```

Comment:
The purpose of `match` and `mismatch` here is to provide an alternative scoring system. So I think you want something like

```
function wscore(base1::Char, base2::Char; match=1, mismatch=-1)
    if base1 == base2
        return match
    end
    # ... etc
end
```